README.md

```diff
@@ -13,14 +13,10 @@ If you want bindings in other languages, feel free to open a github issue. Alter
 
 # Documentation
 
-## Protocol Buffer/gRPC 
+We've got pretty extensive documentation located in the [doc](https://rtradeltd.github.io/TxPB/doc/README.md) folder.
```

Comment:
link is 404

Reply:
This link will only work when the code is merged through.